plugins/usdc_kraken/usdc_kraken_test.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/stretchr/testify/require"
+	"log"
+	"testing"
+)
+
+func TestNewKrakenClient(t *testing.T) {
+	client := NewKrakenClient(&defaultConfig)
+	prices, err := client.FetchPrice([]string{"USDC-USD"})
+	require.NoError(t, err)
+	log.Println(prices)
```

Comment:
See previous comment.

Reply:
Resolved in 4d25fa7c661e76eeaac09170f91d4d6d438fd7ea